tools/AutoTuner/installer.sh

```diff
@@ -6,6 +6,7 @@ cd $script_dir
 
 # Define the virtual environment name
 venv_name="autotuner_env"
+required_version="3.8"
```

Comment:
Didn't we agree somewhere to drop the specific version and keep with either `python` or `python3`?

Reply:
`python` did not resolve to any command in previous iteration, maybe I will do 

```
required_version="3"
python${required_version} ...
```